### ISSUE TITLE
Title Slide missing

### ISSUE THREAD

**Issue by shirou** [tell-k/sphinxjp.themes.gopher#1]
Sorry I have not dig it, but I think a title slide is missing somehow. HTML includes a title node but not displayed.

```
Sphinx==1.6.5
sphinxjp.themes.gopher==0.1.0
```

Tested `index.rst` is below.

```
=====================================================
Sphinxjp.themes.gopher
=====================================================

| tell-k
| https://github.com/tell-k/sphinxjp.themes.gopher

Title Slide
====================================

Why this slide displayed at first?
```

Thanks!

**Comment by tell-k** [comment]
@shirou 

Sorry for the too late reply. 😢 

I've just fixed the bug. and released v0.1.1
Please feel free to try it.

Thanks for your help.
 
https://pypi.python.org/pypi/sphinxjp.themes.gopher/0.1.1

**Comment by shirou** [comment]
Thank you!
